app.py

```diff
@@ -79,6 +80,15 @@ def refresh():
 
     return jsonify(f'The datasets updated successfully! The current number of available datasets is {len(masader)}.')
 
+@app.route('/datasets/<int:index>/issue')
```

Comment:
Could you please make it `issues`? While designing APIs, resources paths should be plural.